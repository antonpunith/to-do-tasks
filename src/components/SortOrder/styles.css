.sort-bar {
  padding: 1rem;
  text-align: right;
}
