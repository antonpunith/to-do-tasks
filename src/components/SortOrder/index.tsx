import React, { FC } from "react";
// styles
import "./styles.css";

const SortOrder: FC<{ sortTask: (name: string) => void }> = props => {
  const { sortTask } = props;
  return (
    <div className="sort-bar">
      <button onClick={() => sortTask("name")}>Sort by Name</button>
      <button onClick={() => sortTask("priority")}>Sort by Priority</button>
    </div>
  );
};

export default SortOrder;
