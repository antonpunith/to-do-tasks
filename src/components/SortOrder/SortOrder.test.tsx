import React from "react";
import { render, cleanup } from "@testing-library/react";
import SortOrder from "./index";

afterEach(cleanup);

it("renders Total component", () => {
  const { asFragment } = render(<SortOrder sortTask={() => {}} />);
  expect(asFragment()).toMatchSnapshot();
});
