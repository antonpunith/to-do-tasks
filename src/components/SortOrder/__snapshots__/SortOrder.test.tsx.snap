// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Total component 1`] = `
<DocumentFragment>
  <div
    class="sort-bar"
  >
    <button>
      Sort by Name
    </button>
    <button>
      Sort by Priority
    </button>
  </div>
</DocumentFragment>
`;
